sample_cgfs/rv32i_priv.cgf

```diff
@@ -1,21 +1,413 @@
 ecall:
   config: 
     - check ISA:=regex(.*I.*); def rvtest_mtrap_routine=True 
-  opcode: 
+  mnemonics: 
     ecall: 0
 
 ebreak:
   config: 
     - check ISA:=regex(.*I.*); def rvtest_mtrap_routine=True 
-  opcode: 
+  mnemonics: 
     ebreak: 0
+    
+ 
+misa_check_combination:
+  config: 
+    - check ISA:=regex(.*I.*Zicsr.*F.*D); 
+  csr_comb:
+    (misa & 0x08) | (misa & 0x20) == 0 : 0			# setting “F”=0 and “D”=1 should results in both “F” and “D” being cleared.
+
+
+misa_mxl_check:
+  config: 
+    - check ISA:=regex(.*I.*Zicsr.*); 
+  csr_comb:
+    misa >> (xlen-2) == 0x01 : 0				# A coverpoint which checks whether the mxl field of misa register is 1.
+
+
+mstatus_Smode:    
+  config: 
+    - check ISA:=regex(.*I.*Zicsr.*); 
```

Comment:
regex needs to include "S" to indicate dependency on Supervisor

Reply:
`check ISA:=regex(.*I.*Zicsr.*S.*);`

Reply:
Added